include/zephyr/usb/class/usb_msc.h

```diff
@@ -46,64 +46,139 @@
 #ifndef ZEPHYR_INCLUDE_USB_CLASS_USB_MSC_H_
 #define ZEPHYR_INCLUDE_USB_CLASS_USB_MSC_H_
 
-/** MSC Subclass and Protocol Codes */
-#define SCSI_TRANSPARENT_SUBCLASS	0x06
-#define BULK_ONLY_TRANSPORT_PROTOCOL	0x50
+/**
+ * @name MSC Subclass and Protocol Codes
+ * @{
+ */
+#define SCSI_TRANSPARENT_SUBCLASS	0x06	/**< SCSI transparent command set sub-class */
```

Comment:
Can we use the same style in the file? (This also better shows how unnecessary these comments are.)
```
/** SCSI transparent command set sub-class */
#define SCSI_TRANSPARENT_SUBCLASS	0x06
```

Reply:
the thinking was to not unecessarily space out the various defines by having the comments on the same line. All defines have a consistent style across the file, I think?